### PR TITLE
Azure Resource Mover Auto-Generated cmdlets

### PR DESCRIPTION
**Pull request by karthik-mut**
## Description

Azure Resource Mover Auto-Generated cmdlets

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
Yes
- [x] The title of the PR is clear and informative
Yes
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
Yes
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
It's a new service
- [x] If applicable, the changes made in the PR have proper test coverage
It's a new service with only auto-generated cmdlets
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
Review is in open state
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
I think this step is not required for generated cmdlets as they are getting generated from autorest tool.

### CODE REVIEW

File: src/ResourceMover/docs/Az.ResourceMover.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+### [Get-AzResourceMoverMoveResource](Get-AzResourceMoverMoveResource.md)
+Gets the Move Resource.
+
+### [Get-AzResourceMoverUnresolvedDependency](Get-AzResourceMoverUnresolvedDependency.md)
+Gets a list of unresolved dependencies.
+
+### [Invoke-AzResourceMoverCommit](Invoke-AzResourceMoverCommit.md)
+Commits the set of resources included in the request body.
+The commit operation is triggered on the moveResources in the moveState 'CommitPending' or 'CommitFailed', on a successful completion the moveResource moveState do a transition to Committed.
```
Comment (natarajks) [inline src/ResourceMover/docs/Az.ResourceMover.md:28]: transition to 'Delete Source' pending

File: src/ResourceMover/Az.ResourceMover.psd1 [hunk 2]
```diff
@@ -0,0 +1,24 @@
+@{
+  GUID = '97d87543-8eef-4574-a70d-7317ec4abe54'
+  RootModule = './Az.ResourceMover.psm1'
+  ModuleVersion = '1.0.0'
```
Comment (dingmeng-xue) [inline src/ResourceMover/Az.ResourceMover.psd1:4]: The initial version should be 0.1.0

Comment (kattamudi-karthik) [reply src/ResourceMover/Az.ResourceMover.psd1:4]: Updated the version.

Comment (dingmeng-xue) [reply src/ResourceMover/Az.ResourceMover.psd1:4]: I cannot see the change. Did you push your commit?

File: src/ResourceMover/examples/Get-AzResourceMoverMoveCollection.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
```
Comment (dolauli) [inline src/ResourceMover/examples/Get-AzResourceMoverMoveCollection.md:1]: Please add related examples in the examples folder. And the examples in docs folder are generated based on the examples you provide here.

File: src/ResourceMover/readme.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+## Run Generation
+In this directory, run AutoRest:
+> `autorest`
+ 
+---
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+> Metadata
```
Comment (dolauli) [inline src/ResourceMover/readme.md:49]: For common configuration, you should require the readme.azure.noprofile.md. See https://github.com/Azure/azure-powershell/blob/generation/src/Databricks/readme.md for details,

File: src/ResourceMover/tools/Resources/.gitattributes [hunk 5]
```diff
@@ -0,0 +1 @@
+* text=auto
```
Comment (dolauli) [inline src/ResourceMover/tools/Resources/.gitattributes:1]: The tools folder is not required to be checked in

**Review by dolauli — CHANGES_REQUESTED** [review]
Some changes are required

File: src/ResourceMover/examples/Get-AzResourceMoverMoveCollection.md (hunk 3 above)
Comment (kattamudi-karthik) [reply src/ResourceMover/examples/Get-AzResourceMoverMoveCollection.md:1]: In this case, can we remove the examples from the doc folder and check-in the docs folder? Also, how are they generated? Can we generate them using an offline process ?

File: src/ResourceMover/Az.ResourceMover.psd1 (hunk 2 above)
Comment (kattamudi-karthik) [reply src/ResourceMover/Az.ResourceMover.psd1:4]: Not sure why it is not showing here. But I am able to see the commit and in the individual file, I am able to see the change.

File: src/ResourceMover/examples/Get-AzResourceMoverMoveCollection.md (hunk 3 above)
Comment (kattamudi-karthik) [reply src/ResourceMover/examples/Get-AzResourceMoverMoveCollection.md]: Added the examples

File: src/ResourceMover/readme.md (hunk 4 above)
Comment (kattamudi-karthik) [reply src/ResourceMover/readme.md:49]: Updated the common configuration .

File: src/ResourceMover/tools/Resources/.gitattributes (hunk 5 above)
Comment (kattamudi-karthik) [reply src/ResourceMover/tools/Resources/.gitattributes]: Removed the tools folder.

**Review by dolauli — APPROVED** [review]
Looks good

**Pull request merged**